frameworks/kafka/tests/client.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, service_options: dict):
         self._service_name = service_options["service"]["name"]
 
     def get_zookeeper_endpoint(self) -> str:
-        zookeeper_str = sdk_networks.get_endpoint(self._package_name, self._service_name, "zookeeper", json=False)
+        zookeeper_str = sdk_networks.get_endpoint_string(self._package_name, self._service_name, "zookeeper")
         return zookeeper_str.strip()
```

Comment:
Could remove the strip call here since get_endpoint_string is doing it automatically